xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -2239,7 +2239,7 @@ admin.colortheme.wikiSetting=The color theme configured at the wiki level is {0}
 admin.colortheme.manage=Manage color themes\u00BB
 admin.customize=Customize
 admin.save=Save
-admin.defaultwikinotinstalled_useflavor=Your wiki seems empty. You may want to use the Extension Manager to install a flavor, it will bring you a lot of features: user profiles, recent activity, administration pages and many more.
+admin.defaultwikinotinstalled_useflavor=Your wiki seems empty. <a href="{0}">You may want to install a flavor</a>, it will bring you a lot of features: user profiles, recent activity, administration pages and many more.
```

Comment:
The link on "install a flavor", not on "you may..."

Reply:
I though it was wrong to have HTML syntax in translations.

Reply:
text before {0}Link label{1} text after

$services.localization.render(keyid,[param1, param2])
$services.localization.render('admin.defaultwikinotinstalled_useflavor',["<a href='$URL'>", '</a>'])